apps/client/src/pages/CheckerPage.js

```diff
@@ -86,6 +90,25 @@ const CheckerPage = ({ checker, topic, resetChecker }) => {
       : console.error(
           `goToQuestion(): ${value} is not an integer, 'next' or 'prev'`
         );
+    const question = checker.stack[questionIndex];
+    // EventName is the question text in lowercase combined with the button that is pressed. (next, prev)
+    const eventName = `${question.text.toLowerCase()} - ${
+      value === "prev" ? eventNames.PREV_QUESTION : eventNames.NEXT_QUESTION
+    }`;
+
+    if (Number.isInteger(value)) {
+      trackEvent({
+        action: actions.CLICK_INTERNAL_NAVIGATION,
+        category: name.toLowerCase(),
+        name: `${question.text.toLowerCase()} - ${eventNames.EDIT_QUESTION}`,
+      });
+    } else {
+      trackEvent({
+        action: actions.CLICK_INTERNAL_NAVIGATION,
+        category: name.toLowerCase(),
+        name: eventName,
+      });
+    }
```

Comment:
I find it strange to make two separate trackEvents where only the `name` is different. It would be better to refactor like this:

```
const name = `${question.text.toLowerCase()} - ${Number.isInteger(value) ? eventNames.EDIT_QUESTION : value === "prev" ? eventNames.PREV_QUESTION : eventNames.NEXT_QUESTION}`

trackEvent({
        action: actions.CLICK_INTERNAL_NAVIGATION,
        category: name.toLowerCase(),
        name,
});

```

(I'm typing this in GH, so I don't know if this code is 100% correct)